### PR TITLE
Liviicc 2289

### PR DESCRIPTION
**Pull request by mialu**
Should this return json also in error situation? Now it's returning compiler's error number and message.

### CODE REVIEW

File: custom/scripts/fileTimestamp.conf [hunk 1]
```diff
@@ -0,0 +1 @@
+UserParameter=fileTimestamp[*],cscript c:\zabbix\scripts\fileTimestamp.vbs //nologo $1
```
Comment (sapajune) [inline custom/scripts/fileTimestamp.conf]: Is this file necessary or is it accidental duplicate of same file under conf?

File: custom/scripts/fileTimestamp.vbs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+If Err.Number <> 0 Then
+	'if path doesn't exist, returns 0
+    WScript.StdOut.Write 0
+    WScript.Quit
+End If
+
+Set colFiles = objFolder.Files
+
+'goes through given folder's subfolders and hunts for the latest timestamp
+ShowSubFolders objFSO.GetFolder(startFolder)
```
Comment (sapajune) [inline custom/scripts/fileTimestamp.vbs]: Could use objFolder variable retrieved earlier in call.

File: custom/scripts/fileTimestamp.vbs [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+Set objFolder = objFSO.GetFolder(startFolder)
+'checking that the given path exists
+If Err.Number <> 0 Then
+	'if path doesn't exist, returns 0
+    WScript.StdOut.Write 0
+    WScript.Quit
+End If
+
+Set colFiles = objFolder.Files
```
Comment (sapajune) [inline custom/scripts/fileTimestamp.vbs]: Looks like unused variable.

File: custom/scripts/fileTimestamp.vbs (hunk 2 above)
Comment (sapajune) [reply custom/scripts/fileTimestamp.vbs]: The script does not check for files in start folder, only in its subfolders.

File: custom/scripts/fileTimestamp.vbs [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+On Error Resume Next
+
+Set objFSO = CreateObject("Scripting.FileSystemObject")
+startFolder = Replace(WScript.Arguments(0),"/","\")
+'startFolder = "f:\startFolder" 'for testing
+
+Set objFolder = objFSO.GetFolder(startFolder)
+'checking that the given path exists
+If Err.Number <> 0 Then
+	'if path doesn't exist, returns 0
```
Comment (sapajune) [inline custom/scripts/fileTimestamp.vbs]: Different indentation.

File: custom/scripts/fileTimestamp.vbs [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+Set dateTime = CreateObject("WbemScripting.SWbemDateTime")
+dateTime.SetVarDate(latestTime)
+
+'writes the latest timestamp in unix time (seconds from 1.1.1970)
+WScript.StdOut.Write DateDiff("s", "1/1/1970", dateTime.GetVarDate (false))
+
+'Method that loops given folder's subfolders recursively
+Sub ShowSubFolders(Folder)
+
+    Set objFSO = CreateObject("Scripting.FileSystemObject")
```
Comment (sapajune) [inline custom/scripts/fileTimestamp.vbs]: Could probably use globally declared FileSystemObject to avoid creating multiple objects.

File: custom/scripts/fileTimestamp.conf (hunk 1 above)
Comment (mialu) [reply custom/scripts/fileTimestamp.conf]: Not necessary and accidental duplicate.

File: custom/scripts/fileTimestamp.vbs (hunk 3 above)
Comment (mialu) [reply custom/scripts/fileTimestamp.vbs]: Yes. Removed.

File: custom/scripts/fileTimestamp.vbs (hunk 4 above)
Comment (mialu) [reply custom/scripts/fileTimestamp.vbs]: Ok, one tabulator was still left. Fixed.

File: custom/scripts/fileTimestamp.vbs (hunk 2 above)
Comment (mialu) [reply custom/scripts/fileTimestamp.vbs]: Yes, changed now to use objFolder.
Yes, it doesn't check for files in start folder and I understood it's ok.

File: custom/scripts/fileTimestamp.vbs (hunk 5 above)
Comment (mialu) [reply custom/scripts/fileTimestamp.vbs]: Ok, changed that.

File: custom/scripts/fileTimestamp.vbs (hunk 2 above)
Comment (kompa3) [reply custom/scripts/fileTimestamp.vbs]: For ticket LIVIICC-2312, I presume it should also check files right in the start folder
